src/traverse.rs

```diff
@@ -259,28 +270,130 @@ mod tests {
             super::to_markdown(&node, "docs", "testuser/index"),
             r#"-- markdown:
 
-- [`a`](testuser/index/a)
-  - [`b`](testuser/index/a/b)
-    - [`c`](testuser/index/a/b/c)
-      - [`d`](testuser/index/a/b/c/d)
-        - [`e`](testuser/index/a/b/c/d/e)
-          - [`f.txt`](testuser/index/a/b/c/d/e/f.txt)
-      - [`readme.md`](testuser/index/a/b/c/readme.md)
+- [`a/`](/testuser/index/a)
+  - [`b/`](/testuser/index/a/b)
+    - [`c/`](/testuser/index/a/b/c/readme.md)
```

Comment:
Use `README` everywhere, and write specific test that explicitly checks for lowercase (lowercase is optional, in happy case I want to see how people write this file in 99% of the cases).